src/app/design/frontend/base/default/template/payment/form/omisecc.phtml

```diff
@@ -117,7 +132,7 @@
             </select>
         </div>
 
-        <input data-omise="omise_token" type="hidden" name="payment[omise_token]">
+        <input id="<?php echo $_code ?>_token" data-omise="omise_token" type="hidden" name="payment[omise_token]">
```

Comment:
@nimid at first I don't want to touch any html elements much (just solve jQuery issue).
But, agree, will remove it. 👍